runtime/gc_glue_java/ScavengerDelegate.cpp

```diff
@@ -345,14 +345,20 @@ MM_ScavengerDelegate::scanContinuationNativeSlots(MM_EnvironmentStandard *env, o
 	bool shouldRemember = false;
 
 	J9VMThread *currentThread = (J9VMThread *)env->getLanguageVMThread();
-	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr)) {
+	if (VM_VMHelpers::needScanStacksForContinuation(currentThread, objectPtr, true)) {
 		StackIteratorData4Scavenge localData;
 		localData.scavengerDelegate = this;
 		localData.env = env;
 		localData.reason = reason;
 		localData.shouldRemember = &shouldRemember;
+		/* In STW GC there are not no racing carrier threads doing mount and no need for the synchronization.
+		 *
+		 * (in STW, unless it's somehow already prevented, but in theory could still be 2+ GC racing threads, but that is not a correctness problem,
```

Comment:
The part in oval brackets of my original comment was directed to you. Not meant to really leave that part in the code (well, at least not the part 'you don't really need to elaborate on that' :) )